doc/access.md

```diff
@@ -22,6 +22,27 @@ of members is [here][Build WG Members].
 A subsection of build members have access to infra machines
 (`infra-*`). The list of members is [here][Infra Admins].
 
+The infra group also have access to:
+
+#### Servers:
+- [DigitalOcean Droplets][] (individual accounts)
+- [Joyent][]
+- [MacStadium][]
+- [Packet.net][] (individual accounts)
+- [Rackspace][]
+- [Scaleway][]
+- [SoftLayer][] (individual accounts)
+- [linuxOne][]
+
+#### Certificates
+- [Apple][]
+- [Digicert for Authenticode][]
+- [GoGetSSL for SSL][]
+
+#### Other
+- [Cloudflare][]
```

Comment:
Just a note here, this is using my email address but we should change it to an @iojs.org account, it sends 2fa tokens via email when you log in with a new IP and timezone differences make that difficult across the team to coordinate. It's not a trivial change, however, as we're using the CF API in a couple of places and it uses the email address as account key iirc. So ... don't let me forget this is something we need to do.

Reply:
https://github.com/nodejs/build/pull/995#discussion_r159559596